tasks/finger_tapping_task/client_finger_tapping_task.py

```diff
@@ -75,7 +75,7 @@ def run(self):
                         color = COLOR_FOREGROUND if state else COLOR_DIM
                         subject = PlayerSquare((main_player_coordinate[0] - other_player_width_offset, other_player_height), color)
                         all_sprites_list.add(subject)
-                        player_counter += 1
+                        player_counter += 0
```

Comment:
Why do we not update player_counter here? Without it, the two white squares of 2 other players will overlap in the same space (i.e., the else condition below will never be met).

We should keep it as `player_counter += 1`

Reply:
I wonder why I changed that.